<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Grid Tests" duration-ms="3" started-at="2014-11-13T15:22:17Z" finished-at="2014-11-13T15:22:17Z">
    <groups>
    </groups>
    <test name="Grid Connectivity" duration-ms="3" started-at="2014-11-13T15:22:17Z" finished-at="2014-11-13T15:22:17Z">
      <class name="edu.vt.qav.tests.grid.RemoteTest">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)" name="setUp" is-config="true" duration-ms="382" started-at="2014-11-13T15:22:17Z" finished-at="2014-11-13T15:22:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://your.hub.ip.addy]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[MAC,WIN,LINUX,ANDROID,WIN8_1]]>
              </value>
            </param>
          </params>
        </test-method>
        <test-method status="FAIL" signature="GridTest(org.openqa.selenium.WebDriver)" name="GridTest" duration-ms="2" started-at="2014-11-13T15:22:17Z" finished-at="2014-11-13T15:22:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.testng.internal.PoolService.isFinished(PoolService.java:154)
	at org.testng.internal.PoolService.submitTasksAndWait(PoolService.java:107)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1155)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.testng.TestNG.privateMain(TestNG.java:1301)
	at org.testng.TestNG.main(TestNG.java:1265)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="tearDown()" name="tearDown" is-config="true" duration-ms="546" started-at="2014-11-13T15:22:17Z" finished-at="2014-11-13T15:22:18Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
